### ISSUE TITLE
JsonSerDe: pretty print JSON

### ISSUE THREAD

**Issue by wajda** [AbsaOSS/commons#46]
```scala
foo.toPrettyJson
```

**Comment by yruslan** [comment]
The way I usually do it. But it requires Jackson dependency.
```scala
  def prettyJSON(jsonIn: String): String = {
    val mapper = new ObjectMapper()

    val jsonUnindented = mapper.readValue(jsonIn, classOf[Any])
    val indented = mapper.writerWithDefaultPrettyPrinter.writeValueAsString(jsonUnindented)
    indented.replace("\r\n", "\n")
  }
```

**Comment by wajda** [comment]
Fixes in 889614a21142c9d3cdf386e88c86ca1e4a63c16d

Shame on me I hit "Ctrl+Shift+K" too fast and pushed the changes with a wrong comment into a wrong branch which happened to be _master_. Being an admin of the repo GitHub didn't tell me a word and before I realized something is wrong TeamCity has already made a release and published it to a Maven Central - ver 0.0.20 :facepalm: 

Yeah... when it comes to automation in certain cases less could be more...
